git_resource/Tiltfile

```diff
@@ -106,11 +122,16 @@ def _parse_repository_url(url):
         splitter = '@'
 
     if splitter == None:
-        tree = '#master'
+        tree = ''
     else:
         parts = url.split(splitter)
         tree = splitter + parts[-1]  # last item
-        url = splitter.join(parts[:-1])  # everything else
+
+        # Make sure we didn't peel of a scheme://user@host/ url
+        if splitter == '@' and '/' in tree:
+            tree = ''
```

Comment:
this doesn't seem to handle the case of `scheme://user@host/some/path@commit`

Reply:
it's subtle! ya, this only gets the last part